tests/flows/test_vesting.py

```diff
@@ -171,3 +172,143 @@ def test_main(
         web3=web3, address=get_address_of_type(config, "Staking")
     )
     assert side_staking.get_vesting_amount(erc20_token.address) == vesting_amount
+
+    # check if vesting is correct
+    available_vesting_before = side_staking.get_available_vesting(erc20_token.address)
+
+    # advance 100 blocks to see if available vesting increased
+    for _ in range(100):
+        # send 1 wei
+        send_ether(consumer_wallet, ZERO_ADDRESS, 1)
+
+    available_vesting_after = side_staking.get_available_vesting(erc20_token.address)
+    assert (
+        available_vesting_after > available_vesting_before
+    ), "Available vesting was not increased!"
+    tx_hash = side_staking.get_vesting(erc20_token.address, consumer_wallet)
+    tx_result = web3.eth.wait_for_transaction_receipt(tx_hash)
+    vesting_event = side_staking.get_event_log(
+        SideStaking.EVENT_VESTING,
+        tx_result.blockNumber,
+        web3.eth.block_number,
+        None,
+    )
+    assert vesting_event[0].args.datatokenAddress == erc20_token.address
+    assert vesting_event[0].args.publisherAddress == erc20_token.get_payment_collector()
+    assert vesting_event[0].args.caller == consumer_wallet.address
+
+    # 101 blocks passed, vesting started block = tx_receipt.blockNumber (deploy pool tx)
```

Comment:
let's make it 100 blocks for easier calcs (meaning the `range` above can be modified).
I'd also assert the vesting amount at each loop/step, not only at the end.